src/Parser/RestorationShaman/Modules/ShamanCore/ShamanAbilityTracker.js

```diff
@@ -44,6 +48,22 @@ class ShamanAbilityTracker extends AbilityTracker {
     }
 
   }
+
+  suggestions(when) {
+    const healingRain = this.getAbility(SPELLS.HEALING_RAIN_CAST.id);
+    const has4PT20 = this.owner.selectedCombatant.hasBuff(SPELLS.RESTORATION_SHAMAN_T20_4SET_BONUS_BUFF.id);
+    const unbuffedHealingRainsPercentage = has4PT20 && ((healingRain.casts - healingRain.withT20Buff) / healingRain.casts);
+    if (has4PT20) {
+      when(unbuffedHealingRainsPercentage).isGreaterThan(0)
+        .addSuggestion((suggest, actual, recommended) => 
+          suggest(<span><SpellLink id={SPELLS.RESTORATION_SHAMAN_T20_4SET_BONUS_BUFF.id} /> buffed <SpellLink id={SPELLS.HEALING_RAIN_CAST.id} /> can make for some very efficient healing, consider ensure casting them with <SpellLink id={SPELLS.RESTORATION_SHAMAN_T20_4SET_BONUS_BUFF.id} />({formatPercentage(unbuffedHealingRainsPercentage)}% healing rains were casted without <SpellLink id={SPELLS.RESTORATION_SHAMAN_T20_4SET_BONUS_BUFF.id} />).</span>)
```

Comment:
`({formatPercentage(unbuffedHealingRainsPercentage)}% healing rains were casted without <SpellLink id={SPELLS.RESTORATION_SHAMAN_T20_4SET_BONUS_BUFF.id} />).` should be in the `actual` value, not in the suggestion.

The recommended format for suggestions is to explain what went wrong and then suggest an alternative, possibly educating users on how to play.